### PR TITLE
Add approval request support.

### PR DESCRIPTION
**Pull request by keiichi-hikita**

### CODE REVIEW

File: ecl/sss/v1/approval_requests/results.go [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+// to interpret it as an approval request.
+type GetResult struct {
+	commonResult
+}
+
+// DeleteResult is the response from a Delete operation. Call its ExtractErr to
+// determine if the request succeeded or failed.
+type DeleteResult struct {
+	eclcloud.ErrResult
+}
```
Comment (shimisho) [inline ecl/sss/v1/approval_requests/results.go]: The Approval API does not have Delete, but is it necessary to define DeleteResult?

**Review by shimisho — COMMENTED** [review]
ecl/sss/v1/approval_requests/testing/doc.go is missing, is that OK?

File: ecl/sss/v1/approval_requests/results.go (hunk 1 above)
Comment (keiichi-hikita) [reply ecl/sss/v1/approval_requests/results.go]: You are right. Will fix it.

**Comment by keiichi-hikita** [comment]
doc.go is also my mistake. I'll add that file in next push.

**Comment by keiichi-hikita** [comment]
I changed my PR to reflect @shimisho 's  review comments.
Pleas have a look o/

**Review by shimisho — APPROVED** [review]
LGTM:)

**Review by ManabuMiwa — APPROVED** [review]
I've compared those files against other SDP's endpoints.
It looks good to me. :)

**Pull request merged**